src/core/serializationHelpers.h

```diff
@@ -95,7 +96,23 @@ void load( Archive& ar, QString& s, const unsigned int )
 template< class Archive >
 void serialize( Archive& ar, QString& s, const unsigned int version )
 {
-    boost::serialization::split_free( ar, s, version );
+    split_free( ar, s, version );
+}
+
+template< class Archive >
+void serialize( Archive& ar, QUuid& uuid, const unsigned int /*version*/ )
+{
+    ar & make_nvp("data1", uuid.data1);
```

Comment:
Actually, for saving to xml the output produced by the string version would be more compact (only one field in the xml file). But this one is more efficient, while I find the c++ code more straightforward to read than the string version... In the end it doesn't matter which one we use - we could also split and use the string version for xml archives and this version for binary. Note: We don't use xml serialization of QUuid at the moment.